web/src/components/PoiDetails.tsx

```diff
@@ -202,6 +216,39 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({
           </LinkContainer>
         )}
       </DetailSection>
+      {(website || phoneNumber || email) && (
+        <>
+          <Spacer borderColor={theme.colors.poiBorderColor} />
+          <Collapsible title={t('contactInformation')} initialCollapsed direction={direction}>
+            <>
+              {website && (
+                <ContactContainer>
+                  <Marker src={iconWebsite} alt={t('website')} />
+                  <a href={website}>
```

Comment:
Since it was not defined clearly in the requirements, its just optional. But i think the whole item including icon should be clickable

Reply:
I thought so too but then I realised that the icon is not clickable in the app version. I think if we change it here, we should also change it in the app?